x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/discover/discover_timeline_state_integration.cy.ts

```diff
@@ -60,11 +60,9 @@ const TIMELINE_RESPONSE_SAVED_OBJECT_ID_PATH =
   'response.body.data.persistTimeline.timeline.savedObjectId';
 const esqlQuery = 'from auditbeat-* | where ecs.version == "8.0.0"';
 
-// TODO: reuse or remove this tests when ESQL tab will be added
-describe.skip(
+describe(
   'Discover Timeline State Integration',
```

Comment:
We should rename this test as well as the file and some locators to be aligned with the new reality.

Reply:
Done